plugin.video.raiyoyo/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.raiyoyo" name="RAI YoYo" provider-name="codingkoala" version="1.0.0">
+    <requires>
+        <import addon="xbmc.python" version="2.24.0" />
+        <import addon="script.module.phate89" version="1.0.0"/>
+    </requires>
+    <extension library="default.py" point="xbmc.python.pluginsource">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <language>it</language>
+        <platform>all</platform>
+        <summary lang="it">I programmi di RAI YoYo</summary>
```

Comment:
no, problem, just make the changes and push them on top, we will squash them for you later